settings.py

```diff
@@ -1715,6 +1715,7 @@ def _pkce_required(client_id):
     'corehq.messaging.smsbackends.twilio.models.SQLTwilioBackend',
     'corehq.messaging.smsbackends.infobip.models.InfobipBackend',
     'corehq.messaging.smsbackends.amazon_pinpoint.models.PinpointBackend',
+    'corehq.messaging.smsbackends.unicel.models.SQLUnicelBackend',
```

Comment:
Curious why this was added in 97a84c0b75160f509823aa7093210c514ac8becf? It was removed in the previous commit.